### ISSUE TITLE
redis-py 3.4.1 in pypi says it is the latest version, but it is different from github code

### ISSUE THREAD

**Issue by yonghee12** [andymccurdy/redis-py#1310]
Thanks for wanting to report an issue you've found in redis-py. Please delete this text and fill in the template below.  
It is of course not always possible to reduce your code to a small test case, but it's highly appreciated to have as much data as possible. Thank you!

**Version**: What redis-py and what redis version is the issue happening on?

**Platform**: What platform / version? (For example Python 3.5.1 on Windows 7 / Ubuntu 15.10 / Azure)

**Description**: Description of your issue, stack traces from errors and code that reproduces the issue

At first, thank you for offering this library.

Version : redis-py 3.4.1 from pypi
Platform : macos catalina
Description:
- Using redis, I happened to realize that hmset is officially deprecated.
- I wondered why deprecated warning didn't came out, and I wanted to change my code. 
- Searching Documentations, I've found that redis-py already adopted this situation, that I can pass mapping in redis-py's hset function.
- I tried in my local, and it didn't work.
- The reason was that my pypi redis-py 3.4.1 doesn't have recent hset code.

Attachments
- my redis-py version
- redis-py code

**Comment by yonghee12** [comment]
I made a mistake putting images.

1. redis-py version
![Screen Shot 2020-03-24 at 4 44 40 PM](https://user-images.githubusercontent.com/24601847/77400738-ffd4c680-6dee-11ea-866a-fb3255e6da84.png)

2. redis-py code
![Screen Shot 2020-03-24 at 4 44 58 PM](https://user-images.githubusercontent.com/24601847/77400751-05caa780-6def-11ea-8e57-89e2634d32ce.png)

**Comment by yonghee12** [comment]
I accidently closed this issue.

**Comment by araujoyuri** [comment]
I have the same problem, even though the code from github was push 2 months ago and the last release on PyPi was on February.

**Comment by DylanModesitt** [comment]
bump

**Comment by rileymcdowell** [comment]
Another bump. Just ran into this same problem, eventually found my way here after scouring docs and comparing source code that was installed vs what is documented. 

Seems like the main issue here is the de-synchronization between the docs hosted at [redis-py.readthedocs.io](https://redis-py.readthedocs.io/en/latest/index.html#redis.Redis.hset) and the version installed via pip, just as @DylanModesitt  suggests.

**Comment by andymccurdy** [comment]
Two points:

1. I've changed the default docs branch on Read The Docs to "stable", which will point to the latest PyPI version. The badge in the README has been updated to point to the "stable" version in Read The Docs as well. This should keep RTD in sync w/ what's released on PyPI. Thanks for pointing out that they were different.

2. I just released 3.5.0 which includes all the latest changes. The "stable" version in RTD was updated at the time of the release.
